https://git.savannah.gnu.org/cgit/pspp.git/commit/?id=f0ac9253e8b84691cec17fc80422993949a91cc1

From f0ac9253e8b84691cec17fc80422993949a91cc1 Mon Sep 17 00:00:00 2001
From: Friedrich Beckmann <friedrich.beckmann@gmx.de>
Date: Sun, 26 Jun 2022 09:07:18 +0200
Subject: tests/atlocal.in - switched from which to command -v

Sam James reported a possible future build problem due to the
usage of the "which" command:

https://savannah.gnu.org/bugs/?62675

I replace "which" with "command -v" as proposed. Thanks for your
support.

Closes: 62675
--- a/tests/atlocal.in
+++ b/tests/atlocal.in
@@ -71,7 +71,7 @@ alias pspp='pspp --no-statrc'
 # Avoids error messages during tests if $TERM is set to an unknown terminal.
 TERM=; unset TERM
 
-pspp_diff=`which diff`
+pspp_diff=`command -v diff`
 if test X"$RUNNER" = Xwine; then
     diff () {
         $pspp_diff -w  "$@"
cgit v1.1
